projects/extension/src/content/ExtensionMessageRouter.ts

```diff
@@ -112,6 +124,12 @@ export class ExtensionMessageRouter {
       return this.#establishNewConnection(message);
     }
 
+    if (data.action === 'spec') {
+      console.log('data action: ', data.action);
+      console.log('message', message.data)
```

Comment:
Change these to use the `debug` helper (or remove them)

Reply:
Sorry about these stray logs - will be removed